server/src/server/app.py

```diff
@@ -22,49 +28,105 @@
 from chatbot import loadCharacters, process_character_config
 import os
 
+# Track active connections
+active_connections = 0
+MAX_CONNECTIONS_PER_INSTANCE = 10
 
-async def websocket_endpoint(websocket: WebSocket):
-    await websocket.accept()
-
-    browser_receive_stream = websocket_stream(websocket)
-
-    # Load character configuration
-    print("Loading character configuration")
-    character = loadCharacters(os.getenv("CHARACTER_FILE", "rolypoly.json"))[0]
-    personality = process_character_config(character)
-    
-    # Combine base instructions with character config
-    full_instructions = BASE_INSTRUCTIONS.format(
-        character_instructions=personality,
-        character_name=character["name"],
-        adjectives=", ".join(character.get("adjectives", [])),
-        topics=", ".join(character.get("topics", []))
-    )
-    print("Full instructions:", full_instructions)
-    for tool in TOOLS:
-        print(tool.name)
-
-    
-    agent = OpenAIVoiceReactAgent(
-        model="gpt-4o-realtime-preview",
-        tools=TOOLS,
-        instructions=full_instructions,
-        voice="verse" #"alloy", "ash", "ballad", "coral", "echo", "sage", "shimmer", and "verse"
-    )
 
-    await agent.aconnect(browser_receive_stream, websocket.send_text)
+async def websocket_endpoint(websocket: WebSocket):
```

Comment:
There are no real changes to the `websocket_endpoint` method - this is just auto-formatting changes (we should add an autoformatter to this repo).